exercises/sublist/Hints.md

```diff
@@ -0,0 +1,4 @@
+## Hints
+This exercise requires you to compare sets of data. In order to do this, you need to use IComparable Interface.
```

Comment:
I would rephrase this to something like: "To be able to compare data, the `IComparable` interface is used."